qiskit/dagcircuit/dagnode.py

```diff
@@ -341,6 +341,10 @@ def name(self, new_name):
         """Sets the Instruction name corresponding to the op for this node"""
         self.op.name = new_name
 
+    def __repr__(self):
+        """Returns a representation of the DAGOpNode"""
+        return f"""DAGOpNode(name : {self.name}, qargs : {self.qargs}, cargs : {self.cargs})"""
+
```

Comment:
In general, `eval(repr(x))` should recreate `x` in situations where that's reasonable.  That means outputting things like `DAGOpNode(name=..., qargs=...)` etc rather than using non-Pythonic colons.

The triple-quotes are unnecessary here because it's only a single line.